src/DGtal/kernel/PointVector.h

```diff
@@ -730,6 +730,17 @@ namespace DGtal
      */
     double norm( const NormType type = L_2 ) const;
 
+    /**
+     * Computes the square L2 norm of a point/vector.
+     * \warning This method performs a conversion
+     * from the type T to double for each components to compute the
+     * norms.
+     *
+     * @return the square norm of the point/vector as a double.
+     */
+    double sqnorm( ) const;
```

Comment:
Is there a reason for choosing the name "sqnorm" instead of "squaredNorm" or "squaredL2Norm" ? Because it is not our usual naming conventions.

Reply:
i've seen this name in previous lib.. But I can switch to squaredNorm is you want

Reply:
let's go with `squaredNorm`